cumulusci/core/config/BaseTaskFlowConfig.py

```diff
@@ -61,3 +63,11 @@ def get_suggested_name(self, name, steps):
             return f'. Did you mean "{match_list[0]}"?'
         else:
             return ""
+
+    @contextmanager
+    def open_project_cache(self, cache_name):
```

Comment:
open_project_cache is a wrapper around project_cache_dir, which we named a month ago.

I think of it as "the cache for the project". In some contexts (e.g. MetaCI flow runner) there is only one project in scope so it can go essentially anywhere writeable, but it is still the cache for the project.

Reply:
But a BaseTaskFlowConfig is not guaranteed to be used in the context of a project.

This is more of a theoretical concern because the `cci task run` and `cci flow run` commands do currently require being used in the context of a project. But I think that's more of an accident than  intentional design. I could imagine wanting to support running tasks/flows defined in the universal config against a global org without being in a project directory.

Reply:
Do you prefer just open_cache ? Not sure what descriptor to put between "open" and "cache".

Reply:
I like that. I think the config object you're calling it on implies something about the intended scope of the cache.